### ISSUE TITLE
poseLandmarker.detectForVideo does not work in Android 13 google chrome (GL ERROR :GL_INVALID_OPERATION : glFramebufferTexture2D: <- error from previous GL command)

### ISSUE THREAD

**Issue by galharth** [google/mediapipe#5100]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Android 13

### Mobile device if the issue happens on mobile device

Pixel 6 API 33 android studio emulator 

### Browser and version if the issue happens on browser

Google Chrome

### Programming Language and version

Javascript

### MediaPipe version

0.9.10

### Solution

Pose

### Describe the actual behavior

poseLandmarkerResults = poseLandmarker.detectForVideo(videoElement, performance.now()); should return poselandmarks

### Describe the expected behaviour

poseLandmarkerResults = poseLandmarker.detectForVideo(videoElement, performance.now()); does not return pose landmarks, it returns empty array []

I get the following errors in the console log:

[.WebGL-0x206a0ce00]GL ERROR :GL_INVALID_OPERATION : glFramebufferTexture2D: <- error from previous GL command

### Standalone code/steps you may have used to try to get what you need

```shell
Install pose landmarks using javascript tasks vision.
Run on google chrome android 13:
poseLandmarkerResults = poseLandmarker.detectForVideo(videoElement, performance.now());

poseLandmarkerResults should include landmarks when the person is inside the video, but instead it returns empty landmarks.
```

**Comment by kuaashish** [comment]
Hi @galharth,

We are currently investigating this matter. In the meantime, could you please try using the latest available version, which is 0.10.9? It appears that you are currently using an older version.

Thank you!!

**Comment by kuaashish** [comment]
Hi @BoHellgren,

We have identified that using an emulator to run this program might not be ideal due to hardware limitations and the program's reliance on specific hardware features for optimal performance. Using a physical device is highly recommended for the best experience and functionality.

Because, Machine learning programs often require direct interaction with real-world hardware resources to function effectively. Emulators, while valuable for development and testing, may not provide the necessary hardware capabilities for this program to run smoothly. Unfortunately, due to limitations, We can not do much about this issue.

Thank you!!
